pgjdbc/src/test/java/org/postgresql/test/extensions/HStoreTest.java

```diff
@@ -66,7 +66,7 @@ public void testHStoreSelectNullValue() throws SQLException {
   }
 
   public void testHStoreSend() throws SQLException {
-    Map<String, Integer> correct = Collections.singletonMap("a", new Integer(1));
+    Map<String, Integer> correct = Collections.singletonMap("a", 1);
```

Comment:
There is a subtle difference since from new Integer() we get always the new object and from autoboxing "1" we get object from integer cache.